sway-lib-std/src/bytes.sw

```diff
@@ -0,0 +1,564 @@
+library bytes;
+
+use ::{alloc::{alloc, realloc}, vec::Vec};
+use ::assert::assert;
+use ::option::Option;
+use ::logging::log;
+use ::intrinsics::size_of_val;
+
+impl raw_ptr {
```

Comment:
Should these be added to the raw_pointer lib itself?

Reply:
![image](https://user-images.githubusercontent.com/59666792/204350867-305eb539-ebc0-41d4-85df-8b54eaec02f5.png)

Reply:
Woah, emphatic!